api/prisma/schema.prisma

```diff
@@ -7,40 +7,204 @@ datasource db {
   showTimeLine  Boolean
+
+  // Nonesense
+  persisted Boolean? @map("__persisted") // Undefined
+  strict    Boolean? @map("__strict") // Undefined
+}
+
+type ProfileUI {
+  isLocked      Boolean? // Undefined
+  showAbout     Boolean? // Undefined
+  showCerts     Boolean? // Undefined
+  showDonation  Boolean? // Undefined
+  showHeatMap   Boolean? // Undefined
+  showLocation  Boolean? // Undefined
+  showName      Boolean? // Undefined
+  showPoints    Boolean? // Undefined
+  showPortfolio Boolean? // Undefined
+  showTimeLine  Boolean? // Undefined
+
+  // Nonesense
+  cachedRelations Json?          @map("__cachedRelations") // Undefined | {}
+  data            ProfileUIData? @map("__data") // Undefined
+  persisted       Boolean?       @map("__persisted") // Undefined
+  strict          Boolean?       @map("__strict") // Undefined
+}
+
+type ProgressTimestamp {
+  timestamp Float
+}
+
+type SavedChallengeFile {
+  contents String
+  ext      String
+  history  String[]
+  key      String
+  name     String
+}
+
+type SavedChallenge {
+  files         SavedChallengeFile[]
+  id            String
+  lastSavedDate Float
 }
 
+model user {
+  id                           String               @id @default(auto()) @map("_id") @db.ObjectId
+  about                        String
+  acceptedPrivacyTerms         Boolean
+  badges                       Json? // Undefined | { coreTeam [][] }
+  completedChallenges          CompletedChallenge[]
+  currentChallengeId           String?
+  donationEmails               String[]             @default([]) // Undefined | String[]
+  email                        String
+  emailAuthLinkTTL             DateTime? // Null | Undefined
+  emailVerified                Boolean
+  emailVerifyTTL               DateTime? // Null | Undefined
+  externalId                   String?
+  github                       String? // Only Null | Undefined
+  githubProfile                String? // Undefined
+  isApisMicroservicesCert      Boolean? // Undefined
+  isBackEndCert                Boolean? // Undefined
+  isBanned                     Boolean? // Undefined
+  isCheater                    Boolean? // Undefined
+  isDataAnalysisPyCertV7       Boolean? // Undefined
+  isDataVisCert                Boolean? // Undefined
+  isDonating                   Boolean
+  isFrontEndCert               Boolean? // Undefined
+  isFrontEndLibsCert           Boolean? // Undefined
+  isFullStackCert              Boolean? // Undefined
+  isGithub                     Boolean? // Undefined
+  isHonest                     Boolean?
+  isInfosecCertV7              Boolean? // Undefined
+  isInfosecQaCert              Boolean? // Undefined
+  isJsAlgoDataStructCert       Boolean? // Undefined
+  isLinkedIn                   Boolean? // Undefined
+  isMachineLearningPyCertV7    Boolean? // Undefined
+  isQaCertV7                   Boolean? // Undefined
+  isRelationalDatabaseCertV7   Boolean? // Undefined
+  isRespWebDesignCert          Boolean?
```

Comment:
```suggestion
  isRespWebDesignCert          Boolean? // Undefined
```
Consistency

Reply:
I really feel like there must have been a reason I did this 🤔